core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/SearchResourceRequest.java

```diff
@@ -212,10 +212,15 @@ public String toString() {
 	
 	@Min(0)
 	private int limit;
+	
+	@Min(0)
+	private int minTermMatch;
```

Comment:
These new fields should be part of the options map instead. The others will be part of the options map later, but for now, please move these new fields inside the option map, you can use a new local OptionKey to access them and provide the same accessors as for the other fields.